contracts/facets/EmbalmerFacet.sol

```diff
@@ -311,6 +356,126 @@ contract EmbalmerFacet {
         return true;
     }
 
+    /// @notice Cancels a sarcophagus. An embalmer may cancel a sarcophagus after
+    /// `initializeSarcophagus` but before `finalizeSarcophagus`. The embalmer's
+    /// fees that were locked up will be refunded.
+    /// @param identifier the identifier of the sarcophagus
+    /// @return The boolean true if the operation was successful
+
+    function cancelSarcophagus(bytes32 identifier) external returns (bool) {
+        // Confirm that the sender is the embalmer
+        if (s.sarcophaguses[identifier].embalmer != msg.sender) {
+            revert LibErrors.SenderNotEmbalmer(
+                msg.sender,
+                s.sarcophaguses[identifier].embalmer
+            );
+        }
+
+        // Confirm that the sarcophagus exists
+        if (
+            s.sarcophaguses[identifier].state !=
+            LibTypes.SarcophagusState.Exists
+        ) {
+            revert LibErrors.SarcophagusDoesNotExist(identifier);
+        }
+
+        // Confirm that the sarcophagus is not already finalized
+        if (bytes(s.sarcophaguses[identifier].arweaveTxId).length > 0) {
+            revert LibErrors.SarcophagusAlreadyFinalized(identifier);
+        }
+
+        // Set the sarcophagus state to done
+        s.sarcophaguses[identifier].state = LibTypes.SarcophagusState.Done;
+
+        address[] memory archaeologistAddresses = s
+            .sarcophaguses[identifier]
+            .archaeologists;
+
+        // Re-calculate the total fees that the embalmer locked up in initializeSarcophagus
+        uint256 storageFees = LibBonds.calculateTotalFees(
```

Comment:
This variable name should probably be `totalFees`, since it includes all fees (not just storage)

Reply:
Good catch. Not sure how that happened